app/src/main/java/com/example/mpvcalculatorrx/mvp/model/MvpModel.java

```diff
@@ -0,0 +1,4 @@
+package com.example.mpvcalculatorrx.mvp.model;
+
+public class MvpModel {
```

Comment:
This class is pointless, if you will use it only to give structure and inheritance to your code, prefer interfaces to avoid waste your extends of your classes.

Reply:
Changed for interface, I was using it to give code structure.